example/lib/custom_code_box.dart

```diff
@@ -168,31 +187,36 @@ class _InnerFieldState extends State<InnerField> {
 
   @override
   Widget build(BuildContext context) {
+    final theme = themes[widget.theme];
+
     return Container(
-      color: _codeController!.theme!['root']!.backgroundColor,
+      color: theme?['root']?.backgroundColor,
```

Comment:
Is there a null-safe way to get the theme?

Reply:
What does look null-unsafe here?

Reply:
If there is no corresponding theme, the Container is left with a (presumably undesired) null color.